models/prices/prices_native_tokens.sql

```diff
@@ -65,6 +65,13 @@ FROM
     ('zec-zcash', null, 'ZEC', null, null),
     ('astr-astar', null, 'ASTR', null, null),
     ('dym-dymension-iou', null, 'DYM', null, null),
-    ('manta-manta-network', null, 'MANTA', null, null)
+    ('manta-manta-network', null, 'MANTA', null, null),
+    ('rose-oasis-network', null, 'ROSE', null, null),
+    ('saga-saga', null, 'SAGA', null, null),
+    ('kas-kaspa', null, 'KAS', null, null),
+    ('egld-elrond', null, 'EGLD', null, null),
+    ('ntrn1-neutron', null, 'NTRN', null, null),
+    ('flr-flare-network', null, 'FLR', null, null),
+    ('ar-arweave', null, 'AR', null, null)
```

Comment:
I am not fully sure which chains are these tokens natively from though.

Reply:
i think we're safe to add, as long as passing tests

Reply:
So, it's all good?
should I update the branch?
because of this warning --> "This branch is out-of-date with the base branch"